packages/types/src/payment-types.ts

```diff
@@ -84,7 +84,7 @@ export enum EVENTS_NAMES {
 export enum ESCROW_EVENTS_NAMES {
   PAID_ESCROW = 'paidEscrow',
   PAID_ISSUER = 'paidIssuer',
-  INITIATED_EMERGENCY_CLAIM = 'initializeEmergencyClaim',
+  INITIATED_EMERGENCY_CLAIM = 'initiateEmergencyClaim',
   REVERTED_EMERGENCY_CLAIM = 'revertEmergencyClaim',
```

Comment:
The enum key is past tense but the value is present tense. Is it intended this way?

Reply:
you got a point, I made all three escrow specific events to be in present tense. the others should stay paid imo because we use paid in other places